src/hist/basehist.py

```diff
@@ -320,7 +325,21 @@ def plot1d(
 
         import hist.plot
 
-        return hist.plot.histplot(self, ax=ax, **_proc_kw_for_lw(kwargs))
+        if self.ndim == 1:
+            return hist.plot.histplot(self, ax=ax, **_proc_kw_for_lw(kwargs))
+        else:
+            if overlay is not None:
+                (cat_ax,) = [ax for ax in self.axes if ax.name == overlay]
+            else:
+                (cat_ax,) = [ax for ax in self.axes if ax.traits.discrete]
+            if cat_ax.traits.discrete:
+                cats = [cat_ax.value(i) for i in range(len(cat_ax))]
+            else:
+                cats = [bh.loc(cat_ax.value(i)) for i in range(len(cat_ax))]
```

Comment:
```suggestion
            cats = list(cat_ax if cat_ax.traits.discrete else cat_ax.centers())
```

Reply:
FYI, to be PlottableHistogram compliant, you'd want to take the averages of the tuples returned by list(cat_ax) if this is not discrete. But we are hist, we know we have more than that.  Also, please use `hist.loc` instead of `bh.loc` if you do need loc inside hist. :)